backendStub/service/FrikkService.js

```diff
@@ -0,0 +1,237 @@
+'use strict';
+
+
+/**
+ * Return data of a specific menu
+ * Returns all the data about the menu with given menuID, even the menu items inside it. The photos of the menu items are saved in the Amazon s3 storage, so the links to the cloud storage are also returned. The frontend will directly download them from the cloud storage, they won't be sent by the backend with this endpoint.
+ *
+ * menuID Integer ID of the menu to be returned
+ * returns MenuRetrievedToCustomer
+ **/
+exports.getMenuCustomer = function(menuID) {
+  return new Promise(function(resolve, reject) {
+    var examples = {};
+    examples['application/json'] = {
+  "restaurant" : {
+    "restaurantName" : "Emilio's Pizza",
+    "address" : "André route 45",
+    "city" : "Rome",
+    "country" : "Italy",
+    "latitude" : 48.4281350802915,
+    "longitude" : -110.0792542197085
```

Comment:
Can we change this to: 

`"address" : " Piazzale Susa",`
 `   "city" : "Milan",`
  `  "country" : "Italy",`
  `  "latitude" : 45.4688346,`
  `  "longitude" : 9.2234114`

Makes no sense to test functionality with random data